src/main/java/no/nav/fo/veilarbregistrering/config/ApplicationConfig.java

```diff
@@ -27,16 +24,13 @@
 @Import({
         ServiceBeansConfig.class,
         DatabaseConfig.class,
-        DataSourceHelsesjekk.class,
```

Comment:
Synes det er mer ryddig at Config for de ulike avhengighetene som database, oppfølging etc også eier sin egen helsesjekk-konfig. Hvis vi skal kvitte oss med en avhengighet eller legge til en ny, bør det gi minst mulig spor her.